apps/web/src/pages/templates/workflow/WorkflowEditorPage.tsx

```diff
@@ -121,6 +125,24 @@ const WorkflowEditorPage = ({
             addStep={addStep}
             setSelectedNodeId={setSelectedNodeId}
           />
+          <When truthy={![StepTypeEnum.DIGEST, null].includes(selectedChannel)}>
```

Comment:
Could we extract this to a variable or something? Hard to follow this statement.

Reply:
I have refactored this now 😄